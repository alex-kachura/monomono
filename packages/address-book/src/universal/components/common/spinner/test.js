import React from 'react';
import { render } from 'react-testing-library';
import { DefaultThemeProvider } from '@beans/theme';
import Spinner from '.';

describe('[Component: Spinner]', () => {
  it('should render correctly', () => {
    const { asFragment } = render(
      <DefaultThemeProvider>
        <Spinner
          mobileStyles="mobile-styles"
          desktopStyles="desktop-styles"
        />
      </DefaultThemeProvider>
    );
    const fragment = asFragment();

    expect(fragment).toMatchSnapshot();
  });
});
