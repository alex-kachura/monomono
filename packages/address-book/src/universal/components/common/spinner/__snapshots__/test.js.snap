// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component: Spinner] should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bdVaJa hyInQo"
  >
    <div
      class="sc-bwzfXH eEqJZu"
    />
  </div>
</DocumentFragment>
`;
